cf-app/src/main/kotlin/space/crowdforce/service/tg/Argument.kt

```diff
@@ -0,0 +1,23 @@
+package space.crowdforce.service.tg
+
+enum class Argument(val argName: String) {
+    OWNED_PROJECT_ID("ownedProjectId"),
+    GOAL_NAME("goalName") {
+        override fun from(map: Map<String, String>): String {
+            return map[rawFrom(map)!!]!!
```

Comment:
looks strange: map[map[key]]. I don't see a reason to redefine the base "from" method.